tests/ellipsoidal_chunk_noncoordinate_parallel.prm

```diff
@@ -86,7 +85,7 @@ subsection Postprocess
 
   subsection Visualization
     set List of output variables 	  = density
-    set Output format                 = gnuplot
+    set Output format                     = gnuplot
```

Comment:
Either do not change the line, or align it. Currently, it is misaligned by one ;-) (or is my browser strange?)